docs/Use_cases/Air_quality_monitoring_use_case.md

```diff
@@ -0,0 +1,229 @@
+# Air quality monitoring - use case
+
+The Coiote DM and Azure IoT Hub integration lets you create custom use cases with data visualization. Follow the tutorial below to see how to set up an inspiringly complex use case in just a few steps.
+
+## Prerequisites
+
+- An active Azure subscription.
+- An active Coiote DM account.
+- An active Microsoft Power BI account.
+- An OpenWeatherMap account with a free API token.
+
+## Creating and configuring an Azure IoT hub and storage account
+
+First you need to add a new IoT hub and a storage account in Azure. Here's how to do it:
+
+### Creating an IoT hub
+
+1. In your Azure portal home view, go to **IoT Hub** and select **Add**.
+![Adding an IoT hub](images/hub_add.png "Add iot hub")
+    - In the **Basics** tab:
+        - select your subscription and resource group,
+        - pick your region,
+        - provide a name for your IoT hub.
+![IoT hub basics tab](images/hub_basics.png "Iot hub basics")
+    - In the **Management** tab:
+        - in **Pricing and scale tier** select,
+        - optionally, turn off **Defender for IoT**.
+    - In the **Review + create** tab, click **Create**.
+![IoT hub management tab](images/hub_create.png "Iot hub management")
+
+### Creating a storage account
+
+While your new IoT hub is deploying, you can add a new storage account:
+
+1. In the Azure portal, go to **Storage accounts** and select **Add**.
+
+   - In the **Basics** tab:
+       - select your subscription and resource group,
+       - provide a name for your storage account,
+       - pick your location.
+   - In the **Review + create** tab, click **Create**.
+
+## Configuring the Azure IoT Hub integration extension
+
+Once the deployments are complete, go to Coiote DM to set up the Azure IoT Hub extension.
+If you haven't done this yet, please follow the [instruction for the Azure IoT Hub integration configuration](../Configuring_Azure_IoT_Hub_integration_extension).
```

Comment:
This link doesn't work. We should consider - should this be a complete tutorial - step by step or we use links to other pages. You wrote the first part - how to create iot hub in Azure here, but you add a link to Configuring the extension in Coiote.

Reply:
If we can shorten the whole tutorial by one step and just provide a link to a clear instruction that is just one click away (and in the same guide), then it is a better option imo. I described how to create an Azure IoT hub because we don't have it covered elsewhere.

Reply:
Ok, that's fine.